README.md

```diff
@@ -27,28 +27,28 @@ Answer: We are ...
 
 ## CV
 
-<details open>
+<details>
 <summary>Can you give me some CV tips?</summary>
 
-### **How to write projects on your CVs:**
+### **How to write projects on your CV:**
 
 - **Summary** (summarise in a line or two what was achieved)
-- **Impact** (Was the result scalable, highly available etc.)
-- **Technologies** (Mention specific cloud or other technologies used - buzzwords)
+- **Impact** (Was the result scalable? highly available? etc)
+- **Technologies** (Mention specific cloud or other technologies used)
 
-This creates a coherent, concise, and dense paragraph for each project. Keep it to no more than 4 lines if you can help it.
+This creates a coherent, concise, and dense paragraph for each project. Keep it to no more than 4 lines.
 
 **Example:**
 
 > 3-Tier Python App Hosted on Azure App Service Deployed w/ Azure DevOps Pipelines
 <br>
-<br> _**Details go here**_
+
 
 --- 
 
 ### **How to write points for your work experience section:**
 
-- **Specific** - Mention a very particular situation/case/ticket/project you worked on: "Configured Multifactor Authentication for 10000+ employees"
+- **Specific** - Mention a very particular situation/case/ticket/project you worked on: "Configured multifactor authentication for 10,000+ employees"
```

Comment:
MFA is a technology so using caps on the M and A seemed fine to me. What's your thoughts? Bad example I used here of course lmao, but maybe we can change the example so people don't get confused and make every starting letter a capital one.

Reply:
Ah yes of course, makes sense it just looked a bit awkward to me